rdtools/degradation.py

```diff
@@ -10,62 +10,59 @@
 import statsmodels.api as sm
 
 
-def degradation_with_ols(normalized_energy):
+def degradation_ols(normalized_energy):
```

Comment:
Oh! One more comment -- I am ok with the function name change, however `degradation_with_ols` is still referenced within `rdtools/__init__.py`. This is so that the function is available in rdtools' namespace. The function reference should be updated to reflect `degradation_ols`.